sos/cleaner/__init__.py

```diff
@@ -685,6 +686,11 @@ def obfuscate_report(self, archive):  # pylint: disable=too-many-branches
                         archive.should_remove_file(short_name)):
                     archive.remove_file(short_name)
                     continue
+                if (self.opts.keep_binary_files and
```

Comment:
Maybe - as the use cases when we "continue" (skip obfuscating the file), so as the use cases grow, it it wort commenting individual tests? Like, I dont know, "skip obfuscation of binary files if --keep-binary-files set" here?

Reply:
Sorry, I'm not sure I understand. You mean we should log that we have skipped the file here?

Reply:
I meant a source code comment for better understanding what `if` branch tests what. Just a thought to improve code understanding. If you feel the conditions are straighforward to understand, no need for it :)

Reply:
You are right, a comment here could help a lot. I'll add something in the next push